pages/manager/PluginHandler.inc.php

```diff
@@ -68,6 +68,21 @@ function plugins($args, &$request) {
 		$templateMgr->assign('helpTopicId', 'journal.managementPages.plugins');
 
 		$site =& $request->getSite();
+		$pluginDir = dirname(dirname(dirname(__FILE__))).'/plugins/';
+		// Don't show the install link unless something can be installed somewhere
```

Comment:
This approach isn't particularly reliable; I'd prefer checking just before an uploaded plugin is installed. That way the check could be made specific and safer.